packages/core/services/fetchComponentData.js

```diff
@@ -60,12 +60,7 @@ export async function cacheResult(endpoint) {
   // Check if we have a cache client set up.
   if (0 === Object.keys(cache.client).length) {
     const result = await fetchComponentData(endpoint);
```

Comment:
This could be simplified.
```suggestion
    log.info('%o', info);
    
    return fetchComponentData(endpoint);
```